src/wob/main.py

```diff
@@ -0,0 +1,211 @@
+        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)"
+                      " Chrome/89.0.4389.90 Safari/537.36"
+    }
+    try:
+        response = requests.get(url, headers=headers)
+    except Exception as e:
+        print("Couldn't Connect To Website.. Try Again.... ", e)
+        return None
+    return response.text
+
+
+def find_element_in_soup(html_string: str, search_element: str, attributes: Dict[str, str],
+                         find_all: bool) -> BeautifulSoup.text:
+    try:
+        soup = BeautifulSoup(html_string, "html.parser")
+        page_title = soup.find("h1", attrs={"class": "title"})
+        if page_title is not None and page_title.text == "Page not found":
+            return "Not Enough Pages"
+        if find_all:
+            searched_element = soup.find_all(search_element, attributes)
+            if not searched_element:
+                return None
+        else:
+            searched_element = soup.find(search_element, attributes)
+    except Exception as e:
+        print("Couldn't Search The Desired Element.... ", e)
+        return None
+    return searched_element
+
+
+def process_info(info_attribute: str, info_value: str, is_book: bool) -> Optional[List[str]]:
+    if is_book:
+        search_list = search_book_info_list()
+        date_attribute = "Year published"
+    else:
+        search_list = search_for_other_than_book()
+        date_attribute = "Release date"
+    if info_attribute in search_list:
+        if info_attribute == date_attribute and len(info_value) != 4 and "-" not in info_value:
+            info_value = info_value[:4] + "-" + info_value[4:6] + "-" + info_value[6:]
+        return [info_attribute, info_value]
+    else:
+        return None
+
+
+def get_book_info_template() -> Dict[str, str]:
+    book_info_dictionary = {
+        "Sku": "",
+        "ISBN 13": "",
+        "ISBN 10": "",
+        "Title": "",
+        "Author": "",
+        "Publisher": "",
+        "Year published": "",
+        "Price": ""
+    }
+    return book_info_dictionary
+
+
+def get_template_other_than_book() -> Dict[str, str]:
+    other_info_dictionary = {
+        "Sku": "",
+        "Title": "",
+        "Studio": "",
+        "EAN": "",
+        "Release date": "",
+        "Price": ""
+    }
+    return other_info_dictionary
+
+
+def get_price_element(html: str) -> Optional[BeautifulSoup]:
+    price_element = find_element_in_soup(html, "div", {"class": "price"}, False)
+    return price_element
+
+
+def get_price_text(html: str) -> Any:
+    price_element = get_price_element(html)
+    if price_element is not None:
+        return price_element.text
+    return ""
+
+
+def get_valid_product_attribute_and_value(product_attribute_value_element) -> List[str]:
+    try:
+        label_text = product_attribute_value_element.label.text
+        div_text = product_attribute_value_element.div.text
+    except Exception:
+        return []
```

Comment:
I would consider still printing out the error here just in case. This can be a nasty way to hide errors. That or reduce the error you're catching